README.md

```diff
@@ -248,6 +248,7 @@ _See https://en.wikipedia.org/wiki/Comparison_of_CalDAV_and_CardDAV_implementati
 
 - [Centrifugo](https://github.com/centrifugal/centrifugo) - Language-agnostic real-time messaging (Websocket or SockJS) server. ([Demo](https://github.com/centrifugal/centrifugo#demo)) `MIT` `Go`
 - [Cherry](https://github.com/rafael-santiago/cherry) - Tiny webchat server. `GPL-2.0` `Go`
+- [Conduit](https://conduit.rs/) - A simple, fast, and reliable chat server powered by [Matrix](https://matrix.org/) and written in Rust. ([Source Code](https://gitlab.com/famedly/conduit)) `Apache-2.0` `Rust`
```

Comment:
please remove the inline link to matrix.org

Reply:
@nodiscc No problem, you may want to update the PR template since there are no guidelines regarding the descriptions other than the character limit.